FormBuilder.cs

```diff
@@ -22,118 +22,43 @@ namespace OpenRA.TilesetBuilder
 {
 	public partial class FormBuilder : Form
 	{
-		string srcfile;
-		int size;
-		public TerrainTypeInfo[] TerrainType;
-		public ImmutablePalette TerrainPalette;
-		public bool PaletteFromImage = true;
-		public string PaletteFile = "";
-		public string ImageFile = "";
-		public int TileSize = 24;
-
-		private static int ColorDiff(Color color, Color curr)
-		{
-			return Math.Abs(color.R - curr.R) + Math.Abs(color.G - curr.G) + Math.Abs(color.B - curr.B);
-		}
+		private string _srcfile;
```

Comment:
Omit the explicit private and please use plain lowercase naming. (See the [Coding Standard](https://github.com/OpenRA/OpenRA/wiki/Coding-Standard).)